nuget/nunitlite.nuspec

```diff
@@ -37,10 +37,15 @@ How to use this package:
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
     <file src="bin/net-2.0/nunitlite.dll" target="lib/net20" />
+    <file src="bin/net-2.0/nunitlite-runner.exe" target="lib/net20" />
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
+    <file src="bin/net-4.0/nunitlite-runner.exe" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
+    <file src="bin/net-4.5/nunitlite-runner.exe" target="lib/net45" />
     <file src="bin/portable/nunitlite.dll" target="lib/dotnet" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/dotnet" />
     <file src="bin/portable/nunitlite.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
```

Comment:
You are right that this is not portable. However, it is the runner to be used to run portable tests from the desktop. In principle, I suppose it should go under tools, but since we have five different flavors of runner, I put it under lib. Do you have any ideas as to how we should distribute this?

BTW, the same question arises with the silverlight runner.